docs/index.md

```diff
@@ -3,4 +3,51 @@ title: "Introduction"
 weight: 10
 ---
 
->This document is under construction.
+The AsyncAPI Generator is a tool that generates anything you want using the **[AsyncAPI file](asyncapi-file.md)** and **[template](template.md)** that are supplied as inputs to the AsyncAPI Generator CLI. The Generator was built with extensibility in mind; you can use the generator to generate anything you want provided that it can be defined in a template, such as code, diagrams, markdown files, microservices, and applications. A number of [community-maintained templates](https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate) are now available for immediate usage.
```

Comment:
```suggestion
The AsyncAPI Generator is a tool that generates anything you want using the **[AsyncAPI document](asyncapi-file.md)** and **[template](template.md)** that are supplied as inputs to the AsyncAPI Generator. The Generator was built with extensibility in mind; you can use the generator to generate anything you want, provided that it can be defined in a template, such as code, diagrams, markdown files, microservices, and applications. A number of [community-maintained templates](https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate) are now available for immediate usage.
```